tsdb_k8s_queries/container-template.json

```diff
@@ -38,13 +32,15 @@
             "name": "metrics"
           },
           {%- endif -%}{# non-serverless-index-settings-marker-end #}
+            {# Note: index settings which can be applied freely in serverless begin here #}
           "mapping": {
             "total_fields": {
               "limit": "10000"
             }
           },
           "query": {
             "default_field": [
+              "message",
```

Comment:
I would say to completely remove it

Reply:
That suggestion was confusing at first to me as I thought the intention was to merge templates described in https://github.com/elastic/rally-tracks/pull/553#issuecomment-1898464136, but then I realised corpus does not include `message` field at all it seems. I've addressed that in https://github.com/elastic/rally-tracks/pull/553/commits/b1bd321591cd7c59dd64798e9c06056b1c2caa43.